packages/trader/src/Modules/MT5/Components/mt5-real-account-display.jsx

```diff
@@ -89,18 +89,18 @@ const MT5RealAccountDisplay = ({
         }
     };
 
-    const onClickFundRealSynthetic = () =>
-        openAccountTransfer(current_list['real.synthetic'], {
+    const onClickFundRealSynthetic = account =>
```

Comment:
Since we are now using a single function to get the key, can we combine these three functions into one and just change type?